src/helpers/utils.js

```diff
@@ -0,0 +1,22 @@
+/* Generic helper functions. */
+
+export function getKeyByValue(object, value) {
```

Comment:
We could probably use some methods from lodash instead of defining our own for these helpers. Here we might use https://lodash.com/docs/4.17.15#findKey

Reply:
@breville I vaguely remember early in the project you had some ideas about not using libraries like lodash? Do you still feel that way?

Reply:
That sounds quite possible, but I don't have strong feelings now :)